.github/workflows/verify_consumer_pacts.yml

```diff
@@ -0,0 +1,200 @@
+name: Verify consumer pacts
+# The purpose of this workflow is to verify ANY consumer contract(s) dependent on BPM provider using Pact framework.
+#
+# This workflow is triggered when
+#
+# 1. Consumer makes a change that results in a new pact published to Pact Broker (will verify ONLY the changed pact and publish the verification results back to the broker)
+# 2. Provider makes a change (runs verification tests against ALL DEPLOYED consumer pact versions and publishes corresponding verification results)
+#
+#
+# The workflow requires Pact broker credentials
+# - PACT_BROKER_USERNAME - the Pact Broker username
+# - PACT_BROKER_PASSWORD - the Pact Broker password
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore: [ '**.md' ]
+  push:
+    branches: [ main ]
```

Comment:
Does this mean it'll run on all merges to main including version bumps? It might be good to include a bump check similar to what we do in a few our other actions.

Reply:
Looking below, it seems that the workflow is a no-op for version bumps. https://github.com/DataBiosphere/terra-billing-profile-manager/pull/227/files#diff-88f8d59210f317b7bdd12e6a807c0b84cf473372221acb3ce860f5a0160183b1R85

Reply:
I added that based on this conversation.